client/src/components/Home.vue

```diff
@@ -14,10 +14,14 @@
             Test your knowledge!
           </p> <br>
           <div v-if="!loggedIn">
-            <a @click="navigateTo('/login')" class="button center is-rounded is-medium">Start </a>
+            <router-link to="/login">
+              <a class="button center is-rounded is-medium">Start </a>
+            </router-link>
           </div>
-          <div v-else-if="loggedIn">
-            <a @click="navigateTo('/topics')" class="button center is-rounded is-medium">Start </a>
+          <div v-else>
+            <router-link to="/topics">
+              <a class="button center is-rounded is-medium">Start </a>
+            </router-link>
```

Comment:
Now that I think about this, this is an ideal case for programmatic navigation. These two router links lead you to a different location according to the state of the user (if it's logged in). Refactor this so that navigateTo receives a name, and data object if needed,  and then decides where to push the user according to his state. And please don't use hardcoded routes for navigation, pass the name and use Component Name to navigate.